test/TestEnvironment.ts

```diff
@@ -0,0 +1,213 @@
+import net from 'net';
```

Comment:
TestEnvironment was a file located at `src/relayclient/TestEnvironment.ts` previously, i moved all the testing code into `test` because it makes more sense

Reply:
makes total sense :+1: